packages/cli/src/shared/constants.ts

```diff
@@ -69,7 +69,7 @@ export const deployTopics = [
 
   'client:init', // legacy
   'action:triggered', // legacy
-] as EventTopic[]
+] as Flatfile.EventTopic[]
```

Comment:
I'm not sure these legacy topics are included in the upgraded cli 😅

Should we update it to include them or is no one relying on them anymore?